pkg/services/accesscontrol/authorize_in_org_test.go

```diff
@@ -70,6 +72,30 @@ func TestAuthorizeInOrgMiddleware(t *testing.T) {
 			acService:       &actest.FakeService{},
 			expectedStatus:  http.StatusForbidden,
 		},
+		{
+			name: "should return 200 when user has permissions for a global org",
+			orgIDGetter: func(c *contextmodel.ReqContext) (int64, error) {
+				return accesscontrol.GlobalOrgID, nil
+			},
+			evaluator:       accesscontrol.EvalPermission("users:read", "users:*"),
+			accessControl:   ac,
+			userCache:       &usertest.FakeUserService{},
+			ctxSignedInUser: &user.SignedInUser{UserID: 1, OrgID: 1, Permissions: map[int64]map[string][]string{0: {"users:read": {"users:*"}}}},
```

Comment:
nit: can we do this to make the test a bit more readable:
```suggestion
			ctxSignedInUser: &user.SignedInUser{UserID: 1, OrgID: 1, Permissions: map[int64]map[string][]string{accesscontrol.GlobalOrgID: {"users:read": {"users:*"}}}},
```